amaechi-chuks/iReporter
# Ft modify UI [#162411972]

**What does this PR do?**
This PR modify UI template as suggested by my LFA(Otse).

**Description of tasks**
- Reduce too many colors on the Landing page
- Validate all input field
- Make all cards consistent by adding border-radius to them
- keep footer down
- Make the login forms standout

**How can this PR be tested**
Open UI folder from this branch and check them out on any browser of your choice

Link to relevant story
#162411972
